metrics/disk/cassandra_kubernetes/cassandra.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+set -x
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+
+source "${SCRIPT_PATH}/../../../.ci/lib.sh"
+source "${SCRIPT_PATH}/../../lib/common.bash"
+test_repo="${test_repo:-github.com/kata-containers/tests}"
+TEST_NAME="${TEST_NAME:-cassandra}"
+cassandra_file=$(mktemp cassandraresults.XXXXXXXXXX)
+
+function remove_tmp_file() {
+	rm -rf "${cassandra_file}"
+}
+
+trap remove_tmp_file EXIT
+
+function cassandra_write_test() {
+	cassandra_start
+	export pod_name="cassandra-0"
+	export write_cmd="/usr/local/apache-cassandra-3.11.2/tools/bin/cassandra-stress write n=1000000 cl=one -mode native cql3 -schema keyspace="keyspace1" -pop seq=1..1000000 -node cassandra"
+ 	number_of_retries="50"
+	for _ in $(seq 1 "$number_of_retries"); do
+		if kubectl exec -i cassandra-0 -- sh -c 'nodetool status' | grep Up; then
+			ok="1"
+ 			break;
+		fi
+ 		sleep 1
+	done
+	sleep 30
```

Comment:
could you document what is this `sleep` for? If this is for having cassandra up, isn't the loop above enough?

Reply:
@chavafg yes we need cassandra up, the loop is not enough to verify that

Reply:
@chavafg changes applied